build_resources.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import subprocess
+import os
+
+
+# Set the Python environment variable. If 'PYTHONPATH' is not set, use the default path "/usr/bin/"
+# or you can override the Python environment path for a specific virtual environment
+PYTHON_ENV = os.getenv("PYTHONPATH", "/usr/bin/")
```

Comment:
What happen if this runs on Windows or MacOS? Are we limiting this script? In that case it should be documented.

Or maybe it's possible to use something like `sys.path`. What do you think?